error calling SprintID: runtime error: invalid memory address or nil pointer dereference with v2.13.0

### Summary 
i'm getting an `invalid memory address or nil pointer dereference` with v2.13.0:

```
[/Users/mblaschke/Projects/webdevops/azure-scheduledevents-manager/main.go:258] - G114 ([gosec] 2022/08/19 09:20:19 template: gosec:9:155: executing "gosec" at <$issue.Cwe.SprintID>: error calling SprintID: runtime error: invalid memory address or nil pointer dereference
make: *** [gosec] Error 1
```

### Steps to reproduce the behavior
run gosec against https://github.com/webdevops/azure-scheduledevents-manager

### gosec version

`2.13.0`

### Go version (output of 'go version')
`go version go1.19 darwin/arm64`

### Operating system / Environment
`macOS` (and on github runners)

### Expected behavior
not failing :)

### Actual behavior
fails

--- a/report/text/template.txt
+++ b/report/text/template.txt
@@ -6,7 +6,7 @@ Golang errors in file: [{{ $filePath }}]:
 {{end}}
 {{end}}
 {{ range $index, $issue := .Issues }}
-[{{ highlight $issue.FileLocation $issue.Severity $issue.NoSec }}] - {{ $issue.RuleID }}{{ if $issue.NoSec }} ({{- success "NoSec" -}}){{ end }} ({{ if $issue.Cwe }} {{ $issue.Cwe.SprintID }} {{ else }} "CWE" {{ end }}): {{ $issue.What }} (Confidence: {{ $issue.Confidence}}, Severity: {{ $issue.Severity }})
+[{{ highlight $issue.FileLocation $issue.Severity $issue.NoSec }}] - {{ $issue.RuleID }}{{ if $issue.NoSec }} ({{- success "NoSec" -}}){{ end }} ({{ if $issue.Cwe }}{{$issue.Cwe.SprintID}}{{ else }}{{"CWE"}}{{ end }}): {{ $issue.What }} (Confidence: {{ $issue.Confidence}}, Severity: {{ $issue.Severity }})
 {{ printCode $issue }}
 
 {{ end }}

--- a/report/text/template.txt
+++ b/report/text/template.txt
@@ -6,7 +6,7 @@ Golang errors in file: [{{ $filePath }}]:
 {{end}}
 {{end}}
 {{ range $index, $issue := .Issues }}
-[{{ highlight $issue.FileLocation $issue.Severity $issue.NoSec }}] - {{ $issue.RuleID }}{{ if $issue.NoSec }} ({{- success "NoSec" -}}){{ end }} ({{ $issue.Cwe.SprintID }}): {{ $issue.What }} (Confidence: {{ $issue.Confidence}}, Severity: {{ $issue.Severity }})
+[{{ highlight $issue.FileLocation $issue.Severity $issue.NoSec }}] - {{ $issue.RuleID }}{{ if $issue.NoSec }} ({{- success "NoSec" -}}){{ end }} ({{ if $issue.Cwe }} {{ $issue.Cwe.SprintID }} {{ else }} "CWE" {{ end }}): {{ $issue.What }} (Confidence: {{ $issue.Confidence}}, Severity: {{ $issue.Severity }})
 {{ printCode $issue }}
 
 {{ end }}

